src/framework/audio/internal/worker/playback.cpp

```diff
@@ -63,6 +63,11 @@ void Playback::deinit()
     disconnectAll();
 }
 
+bool Playback::isInited() const
```

Comment:
You aren't using this method

Reply:
I believe it is useful, so I decided to keep it (after changing the solution when I stopped used it)